src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
```

Comment:
[FUNCTION NAMING] - The constructor of `Herbivore` should not accept a `hidden` parameter, as the `hide` method should be responsible for setting the `hidden` property to `true`.